bigbio/biodatasets/evidence-inference/evidence_inference.py

```diff
@@ -0,0 +1,225 @@
+treatments. Each of these articles will have multiple questions, or 'prompts' associated with them.
+These prompts will ask about the relationship between an intervention and comparator with respect to an outcome,
+as reported in the trial. For example, a prompt may ask about the reported effects of aspirin as compared
+to placebo on the duration of headaches. For the sake of this task, we assume that a particular article
+will report that the intervention of interest either significantly increased, significantly decreased
+or had significant effect on the outcome, relative to the comparator.
+"""
+
+_HOMEPAGE = "https://github.com/jayded/evidence-inference"
+
+_LICENSE = "MIT"
+
+_URLS = {
+    _DATASETNAME: "http://evidence-inference.ebm-nlp.com/v2.0.tar.gz",
+}
+
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+
+_SOURCE_VERSION = "2.0.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+class EvidenceInferenceDataset(datasets.GeneratorBasedBuilder):
+    f"""{_DESCRIPTION}"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="evidence-inference_source",
+            version=SOURCE_VERSION,
+            description="evidence-inference source schema",
+            schema="source",
+            subset_id="evidence-inference",
+        ),
+        BigBioConfig(
+            name="evidence-inference_bigbio_te",
+            version=BIGBIO_VERSION,
+            description="evidence-inference BigBio schema",
+            schema="bigbio_te",
+            subset_id="evidence-inference",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "evidence-inference_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("int64"),
+                    "prompt_id": datasets.Value("int64"),
+                    "pmcid": datasets.Value("int64"),
+                    "label": datasets.Value("string"),
+                    "evidence": datasets.Value("string"),
+                    "intervention": datasets.Value("string"),
+                    "comparator": datasets.Value("string"),
+                    "outcome": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == "bigbio_te":
+            features = schemas.entailment_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        urls = _URLS[_DATASETNAME]
+        data_dir = dl_manager.download_and_extract(urls)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
```

Comment:
The original dataset has defined splits (in `splits/<test/val/train>_article_ids.txt`). Would it be possible to follow the same split?

Reply:
Sure, let me check and update the loader.